pkgs/development/libraries/qt-6/hooks/qtbase-setup-hook.sh

```diff
@@ -75,9 +75,9 @@ else # Only set up Qt once.
     fi
 
     qtPreHook() {
-        # Check that wrapQtAppsHook is used, or it is explicitly disabled.
+        # Check that wrapQtAppsHook/wrapQtAppsNoGuiHook is used, or it is explicitly disabled.
         if [[ -z "$__nix_wrapQtAppsHook" && -z "$dontWrapQtApps" ]]; then
-            echo >&2 "Error: wrapQtAppsHook is not used, and dontWrapQtApps is not set."
+            echo >&2 "Error: neither wrapQtAppsHook nor wrapQtAppsNoGuiHook are used, and dontWrapQtApps is not set either."
```

Comment:
At this point I wonder if this is better written as an imperative. "something something add wrapQtApps(NoGui)Hook or set dontWrapQtApps"?